File: Modul-2/Soal-5/PRAK205-2410817220012-NOOR-KHALISA.py
import math

tinggi = int(input("A: "))
miring = int(input("B: "))

alas = math.sqrt((miring * miring) - (tinggi * tinggi))
keliling = tinggi + alas + miring
luas = (alas * tinggi)/2

print(f"Alas = {int(alas)} cm")
print(f"Tinggi = {int(tinggi)} cm")
print(f"Keliling = {int(keliling)} cm")
print(f"Luas = {int(luas)} cm^2")

File: Modul-2/Soal-5/PRAK205-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
#include <math.h>

int main(){
    int tinggi, miring, alas, keliling, luas;
    printf("A: ");
    scanf("%d", &tinggi);
    printf("B: ");
    scanf("%d", &miring);

    alas = sqrt((miring * miring) - (tinggi * tinggi));
    keliling = tinggi + alas + miring;
    luas = (alas * tinggi)/2;

    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);

    return 0;
}